app/src/main/AndroidManifest.xml

```diff
@@ -27,6 +29,7 @@
 
         <activity
             android:name="com.cyb3rko.flashdim.activities.SettingsActivity"
+            android:theme="@style/AppTheme"
```

Comment:
Is this necessary? As the underlying application already has declared the same theme, I would suppose that this is a duplicated implementation.